.codecov.yml

```diff
@@ -31,7 +31,9 @@ comment:
 
 ignore:
   - "LICENSES"
-  - "*_test.go"       
+  - "*_test.go"
+  - "marker.go"
+  - "failpoint-ctl"
```

Comment:
```suggestion
  - "failpoint-ctl" # Ignore the `failpoint-ctl` command line tool
```

Reply:
But how come this is included in the first place 🤔